app/email.py

```diff
@@ -0,0 +1,88 @@
+from datetime import datetime
+
+users = []
+emails = []
+
+
+class Emails:
+    """class to manipulate emails and users."""
+
+    def __init__(self):
+        self.users = users
+        self.emails = emails
+
+    def add_new_user(self, email, firstname, lastname, password):
+        """add new user."""
+        user = {
+            "id":  len(self.users)+1,
+            "email": email,
+            "firstname": firstname,
+            "lastname": lastname,
+            "password": password
+        }
+        self.users.append(user)
+        return user
+
+    def send_email(self, subject, message, status, sender_id, receiver_id):
+        """add new email."""
+        email = {
+            "email_id":  len(self.emails)+1,
```

Comment:
Please be consistent in naming your variables.
Check out this [resource](https://medium.com/@dasagrivamanu/python-naming-conventions-the-10-points-you-should-know-149a9aa9f8c7)